esmvaltool/recipes/cmorizers/recipe_era5.yml

```diff
@@ -0,0 +1,269 @@
+# ESMValTool
+# recipe_era5.yml
+---
+documentation:
+  description: CMORize ERA5 data
+
+  authors:
+    - andela_bouwe
+    - kalverla_peter
+
+  references:
+    - acknow_project
+
+  projects:
+    - ewatercycle
+
+datasets:
+  - {dataset: ERA5, project: native6, type: reanaly, version: '1', tier: 3, start_year: 1990, end_year: 1990}
+
+preprocessors:
+  daily_mean:
+    daily_statistics:
+      operator: mean
+  daily_min:
+    daily_statistics:
+      operator: min
+  daily_max:
+    daily_statistics:
+      operator: max
+
+diagnostics:
+  hourly:
+    description: CMORize hourly ERA5 data
+    scripts: null
+    variables:
+      orog:
```

Comment:
Could you please make sure the variables are listed in alphabetical order? That avoids duplicates.